editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -382,7 +374,74 @@ export const previewStore: StateCreator<
 
   resultData(flagSet, overrides) {
     const { breadcrumbs, flow } = get();
-    return getResultData(breadcrumbs, flow, flagSet, overrides);
```

Comment:
Decided to simplify this into a single function rather than this previous pattern of `resultData` calling `getResultData` - cleaning up another `reduce` along the way (we aren't even dealing with an _array_ of anything in this method because a single flagset category is passed in :see_no_evil:) !